README.md

```diff
@@ -271,7 +276,7 @@ All configuration fields are required.
 *repo* | The name of the GitHub repository that the bot should serve. | "squid"
 *owner* | The owner (a person or organization) of the GitHub repository. | "squid-cache"
 *dry_run*| A boolean option to enable read-only, no-modifications mode where the bot logs pull requests selected for merging but skips further merging steps, including PR labeling and commit tagging | false
-*staged_run*| A boolean option to enable no-final-modifications mode where the bot performs all the merging steps up to (and not including) the target branch update. Eligible PRs are merged into and tested on the staging branch but are never merged into their target branches. | false
+*staged_run*| A boolean option instructing the bot to avoid target branch updates unless the PR has a human-set `M-cleared-for-merge` label. When enabled, an eligible PR is merged into and tested on the staging branch. A successfully tested staged commit is then automatically merged into its target branch if and only if the PR has an `M-cleared-for-merge` label. | false
```

Comment:
Probably we could describe this simpler if we say something like "the bot ignores this option for PRs marked with M-cleared-for-merge label", i.e., the effect the label really performs.

Reply:
I agree that this could be simplified along the lines you are suggesting. Ignoring the option is slightly imprecise because it is still honored when checking for conflicts among `*_run` options, but I know what you mean.

This change request is no longer applicable because the current PR no longer modifies staged_run functionality, but it was a good comment, thank you.